examples/flutter_gallery/lib/demo/expansion_panels_demo.dart

```diff
@@ -10,8 +10,8 @@ enum _Location {
   Bermuda
```

Comment:
@mpcomplete the changes in this PR for this file attempt to implement the save/cancel buttons (previously they were bogus). It'd be interesting to see how this would look under your patch (it's not great under this one).